### PR TITLE
Improve filter1d usage messages

### PR DESCRIPTION
**Pull request by PaulWessel**
Following the directives in #5341.

### CODE REVIEW

File: src/filter1d.c [hunk 1]
```diff
@@ -185,61 +185,69 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct FILTER1D_CTRL *C) {	/* Deall
...
-	GMT_Message (API, GMT_TIME_NONE, "\t     U: Upper- : Return maximum of all negative points.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   Upper case type B, C, G, M, P, F will use robust filter versions,\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   i.e., replace outliers (2.5 L1 scale (MAD) of median) with median during filtering.\n");
+	GMT_Message (API, GMT_TIME_NONE, "  REQUIRED ARGUMENTS:\n");
+	GMT_Option (API, "<");
+	GMT_Usage (API, 1, "\n-F<type><width>[+l|u]");
+	GMT_Usage (API, -2, "Set filter type.  Choose from convolution and non-convolution filters "
+		"and append filter <width> in same units as time column. "
+		"If <width> is a file (and <type> is not f) it must have a time-series of filter widths. "
+		"Append +h select high-pass filtering [Default is low-pass filtering]. "
```
Comment (KristofKoch) [inline src/filter1d.c:202]:
```suggestion
		"Append +h select high-pass filtering [Default is low-pass filtering].");
```

File: src/filter1d.c [hunk 2]
```diff
@@ -185,61 +185,69 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct FILTER1D_CTRL *C) {	/* Deall
...
+		"Convolution filters:");
```
Comment (KristofKoch) [inline src/filter1d.c:203]:
```suggestion
	GMT_Usage (API, -2, "Convolution filters:");
```

Put the section name in a new line to keep layout similar to `Non-convolution filters:` below.

Comment (PaulWessel) [reply src/filter1d.c]: Done

File: src/filter1d.c (hunk 1 above)
Comment (PaulWessel) [reply src/filter1d.c]: Did not see this but added the missing "to" before select.

**Comment by PaulWessel** [comment]
SHould be OK now?

**Review by KristofKoch — APPROVED** [review]
Looks good!

**Pull request merged**